packages/kit-bg/src/services/ServiceScanQRCode/utils/parseQRCode/handlers/deeplink.ts

```
@@ -7,7 +7,7 @@ import type { IQRCodeHandler, IUrlValue } from '../type';
 // onekey-wallet://search/list?q=onekey
 const deeplink: IQRCodeHandler<IUrlValue> = async (value, options) => {
   const urlValue = options?.urlResult;
-  if (urlValue) {
+  if (urlValue && urlValue.data && urlValue.data.urlSchema) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)